tutorial/deploy_edgex.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 
-docker compose -f docker-compose.edgex.yml up -d
+. ./.env
```

Comment:
Think docker compose grab this from the directory it is run from anywhoooo. but fine to leave it